meta-lmp-base/recipes-graphics/wayland/weston_9.0.0.bbappend

```diff
@@ -0,0 +1,9 @@
+# Weston with Xwayland support (requires X11 and Wayland)
+PACKAGECONFIG[xwayland] = "-Dxwayland=true,-Dxwayland=false,libxcursor"
+
+PACKAGECONFIG_remove_apalis-imx6 = "kms"
```

Comment:
Might need a comment here describing why we're dropping kms for apalis-imx6.

Reply:
Maybe move this change to a file like: `meta-lmp-bsp/recipes-graphics/wayland/weston_%.bbappend` where we can land SoC specific config changes.